examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -55,6 +55,12 @@ CHIP_ERROR PairingCommand::Run(PersistentStorage & storage, NodeId localId, Node
 exit:
     mCommissioner.ServiceEventSignal();
     mCommissioner.Shutdown();
+
+    if (err == CHIP_NO_ERROR)
+    {
+        VerifyOrReturnError(GetCommandExitStatus(), CHIP_ERROR_INTERNAL);
```

Comment:
do the macros inprove readability here? Wondering about:

```cpp
if ((err == CHIP_NO_ERROR) && !GetCommandExitStatus()) {
  return CHIP_ERROR_INTERNAL;
}
```

as it does not use macros and this may make it clearer. Combining exit and return macros in one function has generally not been done before.

Reply:
Yeah, I basically cribbed the style from other command impls...  I would be fine either way here, I think.